src/apps/dictionaries/components/ReleasedVersions.tsx

```diff
@@ -89,6 +90,16 @@ const ReleasedVersions: React.FC<Props> = ({
                         </Tooltip>
                       </CopyToClipboard>
                     </TableCell>
+                      <TableCell>
+                              <Switch
+                                  id={row.id}
+                                  data-testid={row.id}
+                                  checked={row.released}
+                                   // onChange={handleChange}
+                                  name="checkedB"
```

Comment:
May be rename it to `checkedReleaseStatus`

Reply:
Done